r2.json

```diff
@@ -0,0 +1,1326 @@
+[
+	{
+		"key": "key_0",
+		"file": "/Users/vberchet/code/work/opennextjs-cloudflare/examples/e2e/app-router/.open-next/cache/__fetch/QOclhARNsaLkNf40Bkblz/cba55fc763282d057fc465c8ccd55cc0d731ba1471cf2b8f526873a11d42a95b"
+	},
```

Comment:
You probably want to remove this file before merging this.

Reply:
Oops, good catch, thanks Edmund